src/components/checkbox/checkbox.scss

```diff
@@ -98,7 +98,7 @@
     display: inline;
     left: 21px;
     margin-left: -21px;
```

Comment:
It looks like the `left` and `margin-left` properties may be the issue here, doesn't make much sense why they would be overriding each other - and the `left` prop is probably what is throwing the input off.

Would it work if you left `margin-right` as `5px`, changed `margin-left` to `0`, and deleted `left`?

Reply:
I tried this actually and discover the left seemed important for something. Can't remember what. I will have another look.